lib/deaf_grandma.rb

```diff
@@ -5,7 +5,11 @@
 class DeafGrandma
 
   def initialize
-    @bye_counter = 0 
+    @farewell = "SEE YOU LATER SONNY!"
+    @heard_response = "NOT SINCE 1964!"
+    @not_heard_response = "SPEAK UP SONNY!"
+    @bye_limit = 3
```

Comment:
given that these ^^ aren't expected to change, it might be worthwhile to think about constants here, rather than instance variables